devices/pine64-pinetab/kernel/0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch

```diff
@@ -0,0 +1,37 @@
+From b07870e2a05f674acd5e9afd3195b76b52032530 Mon Sep 17 00:00:00 2001
+From: Samuel Dionne-Riel <samuel@dionne-riel.com>
+Date: Sat, 22 Aug 2020 22:50:35 -0400
+Subject: [PATCH] dts: pinephone: Setup default on and panic LEDs
+
+ * The green LED defaults to on.
+ * The red LED is the panic indicator.
+
+The green LED defaults to on for an expected transition through red,
+yellow and green during the different boot stages.
+---
+ arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+index 02d82980cf394..dca546a2c4a09 100644
+--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
++++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
```

Comment:
The pinetab does not have a notification LED, this patch is unneeded.

(But do read further comments too before reacting.)

Reply:
Good catch, patch removed.